src/main/java/baekjoon/barkingdog/linkedList/Silver1158.kt

```
@@ -0,0 +1,53 @@
+package baekjoon.barkingdog.linkedList
+
+fun main() {
+    val (n,k) = readln().split(' ').map(String::toInt)
+    val circularLinkedList = CircularLinkedList<Int>()
+    (1..n).forEach{ circularLinkedList.add(it)}
```

detekt (1.23.8)

[warning] Using the forEach method on ranges has a heavy performance cost. Prefer using simple for loops.

(detekt.performance.ForEachOnRange)